src/screens/generationList/generationList.vue

```diff
@@ -102,14 +102,9 @@ export default {
   methods: {
     loadMore: async function () {
       this.isLoadingMore = true
-      if (this.height - this.numGenerations >= 10) {
-        this.numGenerations += 10
-        await this.$store.dispatch('blocks/getLatestGenerations', Object.keys(this.generations).length + 10)
-      } else if (this.height > this.numGenerations) {
-        let toAdd = this.height - this.numGenerations
-        this.numGenerations += toAdd
-        await this.$store.dispatch('blocks/getLatestGenerations', Object.keys(this.generations).length + toAdd)
-      }
+      const toAdd = Math.min(this.height - this.numGenerations, 10)
```

Comment:
can `this.height - this.numGenerations` be negative? looks I was wrong and `Math.max(Math.min(this.height - this.numGenerations, 10), 0)` should be used in this case.